apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/CollectionAssertionSamples.kt

```diff
@@ -0,0 +1,83 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
+
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.translations.DescriptionComparableAssertion
+import kotlin.test.Test
+
+class CollectionAssertionSamples {
+    private val isLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
```

Comment:
nice idea but one does not see the definition of this in the sample. Thus you need to repeat it in each sample instead. Also, newcomers to Atrium are probably more confused about  `DescriptionComparableAssertion.IS_LESS_THAN.getDefault()` than it helps. Thus I suggest you stick to "is less than" and "is greater than" like we have it in api-fluent